index.html

```diff
@@ -24,71 +24,446 @@
+              target="_blank"
+              rel="noopener noreferrer"
+              ><i class="fa social-media-icon-color fa-linkedin"></i
+            ></a>
+          </li>
+
+          <li class="social-media-icon">
+            <a
+              href="https://medium.com/@haywayaheadshotstrategy.fx"
+              target="_blank"
+              rel="noopener noreferrer"
+              ><i class="fa social-media-icon-color fa-medium"></i
+            ></a>
+          </li>
+
+          <li class="social-media-icon">
+            <a
+              href="https://github.com/Haywayaheadshot"
+              target="_blank"
+              rel="noopener noreferrer"
+              ><i class="fa social-media-icon-color fa-github"></i
+            ></a>
+          </li>
+
+          <li class="social-media-icon">
+            <a
+              href="https://angel.co/u/abubakar-ummar"
+              target="_blank"
+              rel="noopener noreferrer"
+              ><i class="fa social-media-icon-color fa-angellist"></i
+            ></a>
+          </li>
+        </ul>
+      </section>
+      </div>
+    </section>
+    <section class="grid-item">
+      <div class="grid-item-three">
+        <header class="grid-item-three-header">
+          <img
+            src="images/SnapshootPortfolio.svg"
+            alt="A snapshoot Portfolio Header From Figma"
+          />
+        </header>
+        <section>
+          <h2 class="grid-item-two-body-title">Tonic</h2>
+
+          <section class="subhead-title-year">
+            <h4 class="grid-item-two-body-subhead">CANOPY</h4>
+            <img
+              class="counter-image"
+              src="images/Counter.svg"
+              alt="counter shape"
+            />
+            <h6 class="title-year">Front End Dev</h6>
+            <img
+              class="counter-image"
+              src="images/Counter.svg"
+              alt="counter shape"
+            />
+            <h6 class="title-year">2022</h6>
+          </section>
+          <p class="grid-item-two-body-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+
+          <ul class="js-css-html-button-ul">
+            <li class="js-css-html-button-li">
+              <button type="button" class="html-css-js-button">html</button>
+            </li>
+            <li class="js-css-html-button-li">
+              <button type="button" class="html-css-js-button">css</button>
+            </li>
+            <li class="js-css-html-button-li">
+              <button type="button" class="html-css-js-button">
+                javascript
+              </button>
+            </li>
+          </ul>
+
+          <button type="button" class="footer-button">
+            <i class="see-project-text">See Project</i>
+          </button>
+        </section>
+      </div>
+    </section>
+    <section class="grid-item">
```

Comment:
- [Optional] Good job making the work cards and using flexbox to lay out its contents. Please consider wrapping all your work cards in one section and use CSS grid to lay out the contents of this section.

Reply:
@Kamba56 Hi I have noted the changes. I will implement them before merging. Thank you.